nx/test/nx/defn/expr_test.exs

```diff
@@ -198,7 +200,7 @@ defmodule Nx.Defn.ExprTest do
       b = Expr.parameter(nil, {:s, 64}, {2, 2}, 1)
 
       assert Nx.sum(Nx.add(Nx.add(Nx.dot(a, a), Nx.tanh(b)), 2))
-             |> inspect(safe: false) == """
+             |> inspect(safe: false) == s """
```

Comment:
Hi @howard0su! This change should not be necessary. `inspect` should not add `\r\n`. Can you please double check if changing the `inspect` comparison are necessary? Instead, \r\n will be added only when writing to the terminal on Windows.